packages/framework/src/DataFixtures/DemoMultidomain/OrderDataFixture.php

```diff
@@ -56,8 +53,6 @@ public function __construct(
 
     /**
      * @param \Doctrine\Common\Persistence\ObjectManager $manager
-     *
-     * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
```

Comment:
Btw, I'm curious, what way did you find most effective to remove these annotatoins?

Manually? Some cool regular expression or with [`GeneralPhpdocAnnotationRemoveFixer`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/2.11/src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php)? 

I'd probably go lazy manual way ;) but just in case you made some cool scripts, I'd like to see it. More PHP projects would be migrating this way, so this could be very useful.

Reply:
mostly manually but when i saw some similiar parts i created just some simple rules. But still i needed to do it one by one and visual replace.

Reply:
I see, thanks for sharing